client/css/common.css

```diff
@@ -0,0 +1,89 @@
+/* Overwriting Bootstrap CSS */
+.nav-link {
+    color: #333;
+}
+
+.nav-link.active {
+    color: #007bff;
+}
+
+.row {
+    margin-left: 0px !important;
+    margin-right: 0px !important;
+}
+
+/* Top bar */
+#top-bar {
+    padding-bottom: 0px;
+    padding-top: 0px;
+}
+
+/* Sidebar */
+#sidebar {
+    width: inherit;
+    min-width: 220px;
+    max-width: 220px;
+    background-color: #f5f5f5;
```

Comment:
We should probably create a constants file for these colours. (No need for this PR though)